<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Strack Task Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,4,2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Tasks</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="my_task_check">
           <property name="text">
            <string>My Tasks</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="FilterLineEdit" name="task_filter_edit">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="task_page">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="asset_tab">
            <attribute name="title">
             <string>Asset</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QTreeView" name="task_tree"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="shot_tab">
            <attribute name="title">
             <string>Shot</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QTreeView" name="treeView"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="unfinished_check">
           <property name="text">
            <string>unfinished only</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="FilterCombo" name="project_combo">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>project</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="FilterCombo" name="entity_type_combo">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>Asset</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Shot</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="FilterCombo" name="parent_combo">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>sequence</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="FilterCombo" name="entity_combo">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>shot</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="FilterCombo" name="task_combo">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>task</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="area_tab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="work_area">
           <attribute name="title">
            <string>Work Area</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTreeView" name="treeView_2"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pub_area">
           <attribute name="title">
            <string>Publish Area</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTreeView" name="treeView_4"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="info_grp">
          <property name="title">
           <string>Version Info</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>18</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QFrame" name="thumb_frame">
               <property name="minimumSize">
                <size>
                 <width>240</width>
                 <height>135</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>240</width>
                 <height>135</height>
                </size>
               </property>
               <property name="sizeIncrement">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>18</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="info_table"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="todo_grp">
          <property name="title">
           <string>Todos</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QListView" name="todo_list"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterCombo</class>
   <extends>QComboBox</extends>
   <header location="global">cgtk_qt/FilterCombo</header>
  </customwidget>
  <customwidget>
   <class>FilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>cgtk_qt/FilterLineEdit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
